simonfabcic/32_-_NestJS_-_ReactJS-app
# Adding modes `order`, `product` and `order item` in shop

[What?]
Adding models:
- order
- product
- order item

[Why?]
So I can create a purchase

[How?]
Creating models, where `order` and `product` are connected through `orderItem`

[Testing?]
- creating a factory for `Product`
- then testing:
    - add product to order
    - if only one type of `product` can be added to the `order` (IntegretyError)
    - check the order total quantity
    - removing a product from the order
    - multiple orders
